libs/testing/src/jobs.service.ts

```diff
@@ -74,38 +64,29 @@ export class JobsService {
         }
       : {};
 
+    let totalCount = 0;
+
     do {
+      // Wait until Bull queues are empty
       totalCount = (await this.getQueueMetric()).totalCount;
-      runningJobs = await this.jobRepository.count({
-        _organizationId: organizationId,
-        ...typeMatch,
-        ...workflowMatch,
-        status: {
-          $in: [JobStatusEnum.PENDING, JobStatusEnum.QUEUED, JobStatusEnum.RUNNING],
-        },
-      });
-    } while (totalCount > 0 || runningJobs > unfinishedJobs);
+      // Wait until there are no pending, queued or running jobs in Mongo
+      runningJobs = Math.max(
```

Comment:
why we need to do max?

Reply:
I'm not sure if I get it, here we have the result of this.jobRepository.count